app/scripts/contentscript.js

```diff
@@ -928,10 +946,40 @@ function prepDiff(spdxid, time, html, details) {
   if (details.licenseComments) {
     hoverInfo += "&#10;comments: " + _.escape(details.licenseComments);
   }
-  var title = `<a href="https://spdx.org/licenses/${spdxid}.html" target="_blank" title="${hoverInfo}">${details.name} (${spdxid})</a>`;
+
+  let confidenceIndicator = '';
+  if (percentage !== undefined) {
+      let confidenceText = '';
+      let confidenceClass = '';
+      let icon = '';
+
+      if (percentage >= 90) {
+          confidenceText = 'Good Match';
+          confidenceClass = 'full-match';
+          icon = '✔';
```

Comment:
I assume emojis don't have issues with displaying in modern browsers, but just checking.